openhexa/cli/cli.py

```diff
@@ -407,4 +430,4 @@ def _terminate(message: str, exception: Exception = None, err: bool = False):
     click.echo(click.style(message, fg="red"), err=err)
     if settings.debug and exception:
         raise exception
-    click.Abort()
+    sys.exit(1)
```

Comment:
@qgerome  I've restored this the sys call because it seems like  click.Abort doesn't stop the running process, only Click. (I was testing the pipeline_delete function)

Reply:
What is the other process triggered by the CLI ?

Reply:
For example in the case of `delete_pipeline` function : we have a first call to get pipeline  `get_pipeline(pipeline_code)`
if the pipeline doesn't exist it should exit but with click.abort(), it continues and prompt the user to enter the confirmation code.

![image](https://github.com/BLSQ/openhexa-sdk-python/assets/25453621/5d4ca669-0cce-4840-877d-23055848b599)